Content/Items/Gear/Gear.cs

```diff
@@ -18,6 +19,9 @@ internal abstract class Gear : PoTItem
 {
 	private Socketable[] _sockets = []; // [new Imps()];
 	private int _selectedSocket = 0;
+	
+	public virtual string Description => "";

```

Comment:
i dont see a reason why we couldn't give this to all PoTItems

Reply:
Added to PoTItems , and also added the missing tooltip sections that were missed in the merge.